source/reference.txt

```diff
@@ -2,8 +2,11 @@
 Reference
 =========
 
+:ref:`about-mongosync`
+   The primary process used in ``mongosync``.
+
 :ref:`c2c-mongosync`
-   The primary process used in {+c2c-product-name+}.
+   The options and behavior {+c2c-product-name+}.
```

Comment:
Will this page still have behavior?

Reply:
It won't, but I thought I would delete the behavior part once I get to this ticket.